admin_manual/enterprise_user_management/index.rst

```diff
@@ -5,4 +5,5 @@ User Management (ES only)
 .. toctree::
     :maxdepth: 2
 
+    file_firewall
```

Comment:
It could go in user management, because it's about controlling user access, or it could go in file management, or it could go in its own category. I think own category is best.